share/ramble/cloud-build/tutorials/ramble-tutorial-1.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+
+steps:
+  - name: gcr.io/cloud-builders/git
+    args:
+      - fetch
+      - '--unshallow'
+    id: ramble-clone
+  - name: us-central1-docker.pkg.dev/$PROJECT_ID/ramble-repo/ramble-${_BASE_IMG}-${_BASE_VER}-spack${_SPACK_REF}-conda${_CONDA_VER}:latest
+    args:
+      - '-c'
+      - |
+        cd /workspace
+
+        export PATH=$$(. /opt/spack/share/spack/setup-env.sh && spack location -i miniconda3)/bin:$${PATH}
+
+        . /opt/spack/share/spack/setup-env.sh
+        . /workspace/share/ramble/setup-env.sh
+
+        echo "Spack version is $(spack --version)"
+        echo "Python version is $(python3 --version)"
+
+        spack mirror add ci_cache ${_CI_CACHE}
+        spack buildcache keys --install --trust
+
+        set -e
+
+        ramble list
+
+        ramble list h
+
+        ramble list h*
+
+        ramble list -t test-app
+
+        ramble attributes --all --tags
+
+        ramble info hostname
+
+        ramble workspace create -d hello_world -c /workspace/examples/tutorial_1_config.yaml
+
+        ramble workspace activate ./hello_world
+
+        ramble workspace setup
+
+        ramble on
+
+        ramble workspace analyze
+
+        cat hello_world/results.latest.txt
```

Comment:
Do we want to do some assertion on the result?

Reply:
Good suggestion. LMK if what I added looks good.